proseco/core.py

```diff
@@ -332,6 +332,11 @@ def __repr__(self):
         return (f'<{self.__class__.__name__} name={self.name} default={self.default} '
                 f'pickle={self.pickle}>')
 
+class QuatMetaAttribute(MetaAttribute):
+    def __set__(self, instance, value):
+        if not isinstance(value, Quat):
+            value = Quat(value)
```

Comment:
This seems fine.  I've sometimes run into issues with the not-quite-normalized Quat, but hopefully anything that reaches this point will be fine.

Reply:
Default handling of a not-quite-normalized Quat should be handled upstream by Quat.  Maybe auto-fix if within some threshold?  Is there something like that already?

Reply:
Right now there's just a good error message.   For proseco I had the MATLAB side explicitly normalize to avoid the possibility of it being a problem with these attitudes (relying on later checking to catch any problem).